demo/com/tenbilac.cfg

```diff
@@ -6,8 +6,9 @@
 
 [setup]
 
-# Pick a name for your training run (runs with different settings can potentially be mixed later)
-name: Test
+# Pick a name for your training run (e.g. net55). 
+# Leave it blank and the filename of the present config file will be used instead -- this is the recommended use.
+name
```

Comment:
I left it in the official configuration file example.
But ok, yes, I'll add it here as well.